bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/target/env/GuideGroup.scala

```diff
@@ -251,6 +251,11 @@ case class GuideGroup(grp: GuideGrp) extends java.lang.Iterable[GuideProbeTarget
     ps
   }
 
+  lazy val isAutomatic: Boolean = grp match {
```

Comment:
````scala
def isAutomatic: Boolean =
  grp.isAutomatic
````

I don't think we want a `val` for this (`lazy` or otherwise) and we might as well use the corresponding underlying method on `GuideGrp`.  We might as well add `isManual` too.